cernopendata/modules/records/serializers/schemaorg.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Open Data Portal.
+# Copyright (C) 2018 CERN.
+#
+# CERN Open Data Portal is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# CERN Open Data Portal is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with CERN Open Data Portal; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+"""CERN Opendata schema.org JSON-LD serializer."""
+
+
+from invenio_records_rest.serializers.json import JSONSerializer
+
+from cernopendata.modules.records.serializers.schemas \
+    import schemaorg_schemas as schemas
+
+
+class OpendataSchemaorgSerializer(JSONSerializer):
+    """CERN Open Data schema.org serializer.
+
+    Serializes a Record based on it's type (Dataset, Software, etc.) to
+    schema.org compatible JSON-LD syntax.
+    """
+
+    SCHEMAS = {
+        'Dataset': schemas.DatasetSchemaorgSchema,
```

Comment:
If we are worried that we intermix "JSON schemas" and "Marshmallow schemas" here and there in the codebase, so that `git grep` will become hard to read, we can think of inventing e.g. "MSchema" for the latter concept, but I'm not sure it would be worth the editing trouble...